backend/zeno_backend/database/select.py

```diff
@@ -815,7 +835,7 @@ def project_from_uuid(project_uuid: str) -> Project | None:
             [project_uuid],
         )
         if len(project_result) == 0:
-            raise Exception("Project does not exist.")
+            return None
```

Comment:
Exceptions seem like a much better way to communicate the cause of the error compared to just returning None.

Reply:
All of the calling functions expect it to return None. Maybe can do another PR to fix all these functions taht return none?